app/celery/provider_tasks.py

```diff
@@ -229,10 +229,13 @@ def _handle_delivery_failure(  # noqa: C901 - too complex (11 > 10)
         raise NotificationTechnicalFailureException from e
 
     elif isinstance(e, NonRetryableException):
-        if 'opted out' in str(e).lower():
+        if 'opted_out' in str(e).lower() or 'opted out' in str(e).lower():
+            # If sending through PinPointV2, ConflictException with reason DESTINATION_PHONE_NUMBER_OPTED_OUT
+            # will return STATUS_REASON_BLOCKED.
             status_reason = STATUS_REASON_BLOCKED
         else:
             # Calling out this includes that are too long.
```

Comment:
This isn't part of your PR, but I would delete this line.

Reply:
Updated.